lang/cs/Org.Apache.REEF.Common/Org.Apache.REEF.Common.csproj

```diff
@@ -39,14 +39,14 @@ under the License.
     <Reference Include="protobuf-net">
       <HintPath>$(PackagesDir)\protobuf-net.$(ProtobufVersion)\lib\net40\protobuf-net.dll</HintPath>
     </Reference>
+    <Reference Include="System" />
```

Comment:
Is System and System.Core needed here? I see other projects that also include the Reactive dependencies but they dont pull in the System dependencies.